### PR TITLE
[SMALL] support for DateTime.UtcNow

### PR DESCRIPTION
**Pull request by maumar**
based on pull request produced by dcga

### CODE REVIEW

**Comment by maumar** [comment]
@anpete @divega

File: src/EntityFramework.SqlServer/Query/ExpressionTranslators/DateTimeNowTranslator.cs [hunk 1]
```diff
@@ -18,7 +18,14 @@ public virtual Expression Translate([NotNull] MemberExpression memberExpression)
                 && memberExpression.Member.DeclaringType == typeof(DateTime)
                 && memberExpression.Member.Name == nameof(DateTime.Now))
             {
-                return new SqlFunctionExpression("GETDATE", Enumerable.Empty<Expression>(), memberExpression.Type);
+                if (memberExpression.Member.Name == nameof(DateTime.Now))
+                {
+                    return new SqlFunctionExpression("GETDATE", Enumerable.Empty<Expression>(), memberExpression.Type);
```
Comment (anpete) [inline src/EntityFramework.SqlServer/Query/ExpressionTranslators/DateTimeNowTranslator.cs]: Can we add another ctor to SqlFunctionExpression for the no args case?

**Comment by anpete** [comment]
:shipit:

File: src/EntityFramework.SqlServer/Query/ExpressionTranslators/DateTimeNowTranslator.cs (hunk 1 above)
Comment (maumar) [inline src/EntityFramework.SqlServer/Query/ExpressionTranslators/DateTimeNowTranslator.cs]: sure

**Pull request closed without merging**